record/record_test.go

```diff
@@ -102,7 +102,7 @@ var _ = Describe("Out New Relic", func() {
 
 		It("Correctly handles a JSON array in a []interface{}", func() {
 			// Given
-			inputRecord := map[interface{}]interface{}{
+			inputMap := map[interface{}]interface{}{
```

Comment:
I actually meant the opposite: rename inputMap to inputRecord everywhere 🤣 But that's OK, we can merge this code and do it later.